From 3ef3ea938391f618515d5a12ed66638c51fc7313 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Thu, 28 Oct 2021 16:32:28 +0200
Subject: [PATCH] quiche: add missing build unit for ClientProofSource.

After latest change in quiche, ClientProofSource is used and available
in source code, but it was not added in BUILD.gn.

Change-Id: I67eb48a52e08f19e9c8fa694a15157c15247f419
---

diff --git a/net/third_party/quiche/BUILD.gn b/net/third_party/quiche/BUILD.gn
index c63464c..88f3800 100644
--- a/net/third_party/quiche/BUILD.gn
+++ b/net/third_party/quiche/BUILD.gn
@@ -244,6 +244,8 @@
     "src/quic/core/crypto/chacha_base_encrypter.h",
     "src/quic/core/crypto/channel_id.cc",
     "src/quic/core/crypto/channel_id.h",
+    "src/quic/core/crypto/client_proof_source.cc",
+    "src/quic/core/crypto/client_proof_source.h",
     "src/quic/core/crypto/common_cert_set.cc",
     "src/quic/core/crypto/common_cert_set.h",
     "src/quic/core/crypto/crypto_framer.cc",
